tst/EnergyPlus/unit/HVACDXSystem.unit.cc

```diff
@@ -789,7 +789,7 @@ TEST_F(EnergyPlusFixture, VariableSpeedCoils_LatentDegradation_Test)
 
     thisSys->controlCoolingSystemToSP(*state, airLoopNum, FirstHVACIteration, HXUnitOn, CompressorOn);
     SHR = state->dataVariableSpeedCoils->VarSpeedCoil(1).QSensible / state->dataVariableSpeedCoils->VarSpeedCoil(1).QLoadTotal;
-    EXPECT_NEAR(SHR, 0.7624, 0.0001);
+    EXPECT_NEAR(SHR, 0.7626, 0.0001);
```

Comment:
Unit tests show the small change as a result of using StdRhoAir

Reply:
:+1: